app/views/stories/_form.html.erb

```diff
@@ -0,0 +1,71 @@
+<%= form_with(model: @story, url: [@map_asset, @map_asset.stories.build], local:false) do |form| %>
+
+  <div class="modal-body">
+
+    <% if @story.errors.any? %>
+      <div id="error_explanation">
+        <h2><%= pluralize(@story.errors.count, "error") %> prohibited this story from being saved:</h2>
+        <ul>
+          <% @story.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <div class="field form-group">
+      <%= form.label :name %>
+      <%= form.text_field :name, class:"form-control", readonly: true , value: @map_asset.name %>
+    </div>
+
+    <div class="field form-group">
+      <%= form.label :address %>
+      <%= form.text_field :address, class:"form-control", readonly: true, value: @map_asset.address %>
+    </div>
+
+    <div class="field form-group">
+      <%= form.label :category %>
+      <%= form.select :category, ["Art", "Education", "Health", "Business", "Enviroment", "Housing", "Child Care", "Food", "Industry", "Safety", "Transportation"] %>  
+    </div>
+
+    <div class="field form-group" id="rate">
+      <%= form.label :rating %>
+      <div id="rating"></div>
+      <%= form.hidden_field :rating, id: "review_star" %>
+    </div>
+
+    <div class="field form-group">
+      <p id="rating_check" >I want to leave a rating</p>
```

Comment:
Are you sure the text says "I `want` to leave a rating" ?

Reply:
Done